operations/host_spawn.go

```diff
@@ -1375,6 +1375,61 @@ Examples:
 	}
 }
 
+func hostFindWithIpAddress() cli.Command {
+	const (
+		ipAddressFlagName = "ip-address"
+		spruceFlagName    = "spruce"
+	)
+	return cli.Command{
+		Name:  "find-by-ip-address",
+		Usage: "find host using ip address",
+		Flags: addHostFlag(
+			cli.StringFlag{
+				Name:  joinFlagNames(ipAddressFlagName, "ip"),
+				Usage: "ip address used to find host",
+			},
+			cli.BoolFlag{
```

Comment:
This makes spruce optional, whereas spruce should really be the default. Can we call this legacy instead?